cuda_bindings/docs/source/overview.md

```diff
@@ -193,20 +189,23 @@ checkCudaErrors(driver.cuMemcpyHtoDAsync(
 With data prep and resources allocation finished, the kernel is ready to be
 launched. To pass the location of the data on the device to the kernel execution
 configuration, you must retrieve the device pointer. In the following code
-example, `int(dXclass)` retries the pointer value of `dXclass`, which is
-`CUdeviceptr`, and assigns a memory size to store this value using `np.array`.
-
-Like `cuMemcpyHtoDAsync`, `cuLaunchKernel` expects `void**` in the argument list. In
-the earlier code example, it creates `void**` by grabbing the `void*` value of each
-individual argument and placing them into its own contiguous memory.
+example, we call `int(XXclass)` to retrieve the device pointer value for the
+associated XXclass as a Python `int` and wrap it in a `np.array` type.
 
 ```python
-# The following code example is not intuitive 
-# Subject to change in a future release
 dX = np.array([int(dXclass)], dtype=np.uint64)
 dY = np.array([int(dYclass)], dtype=np.uint64)
 dOut = np.array([int(dOutclass)], dtype=np.uint64)
+```
 
+The launch API `cuLaunchKernel` also expects a pointer input for the argument list
+but this time it's of type `void**`. What this means is that our argument list needs to
+be a contiguous array of `void*` elements, where each element is the pointer to a kernel
+argument. Since we already prepared each of our arguments into a `np.array` type, the
+construction of our final contiguous array is done by retrieving the `XX.ctypes.data`
+of each kernel argument.
```

Comment:
```suggestion
be a contiguous array of `void*` elements, where each element is the pointer to a kernel
argument on either host or device. Since we already prepared each of our arguments into a `np.array` type, the
construction of our final contiguous array is done by retrieving the `XX.ctypes.data`
of each kernel argument.
```